package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "marked-emoji",
+  "version": "1.0.0-SNAPSHOT",
+  "description": "Simple plugin to add emoji support",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "fmt": "npx prettier --write index.js test/ package.json",
+    "fmt-check": "npx prettier --check index.js test/ package.json"
```

```suggestion
    "fmt-check": "npx prettier --check index.js test/ package.json",
    "preversion": "npm run fmt-check && npm test"
```

This will automatically run the format check and tests for you when you run the `npm version ...` command (: